src/server/routes.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import fetch from 'node-fetch';
+import domParser from 'dom-parser';
+
+const router = express.Router();
+
+router.get('/search', (_req, res) => {
+  getData('airpods');
+  res.status(200).send('ok');
```

Comment:
I would love to test your API, but there is something missing. 
First, a deployment is missing to test it (thus, I have to clone and run it).
If I call `/api/search`, the `price` is logged on server console.
It would be awesome to get send the price back to the client like:

```suggestion
  const price = await getPrice('airpods');
  res.status(200).json({
    name: 'airpods',
    price: price
  });
```